exporter/datadogexporter/metadata/host.go

```diff
@@ -84,23 +100,20 @@ func HostnameFromAttributes(attrs pdata.AttributeMap) (string, bool) {
 
 	// Kubernetes: node-cluster if cluster name is available, else node
 	if k8sNodeName, ok := attrs.Get(AttributeK8sNodeName); ok {
-		if k8sClusterName, ok := attrs.Get(conventions.AttributeK8sCluster); ok {
-			return k8sNodeName.StringVal() + "-" + k8sClusterName.StringVal(), true
+		if k8sClusterName, ok := getClusterName(attrs); ok {
+			return k8sNodeName.StringVal() + "-" + k8sClusterName, true
 		}
 		return k8sNodeName.StringVal(), true
 	}
 
-	// container id (e.g. from Docker)
-	if containerID, ok := attrs.Get(conventions.AttributeContainerID); ok {
-		return containerID.StringVal(), true
-	}
-
 	// handle AWS case separately to have similar behavior to the Datadog Agent
```

Comment:
I think this comment doesn't apply anymore, given that we have separate logic for AWS, GCP & Azure now